.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,75 @@
+name: CI
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+  schedule:
+    - cron:  '0 5 * * FRI'
+
+jobs:
+  CI:
+    continue-on-error: ${{ matrix.distro == 'ubuntu:devel' || matrix.distro == 'opensuse:latest' || matrix.distro == 'fedora:rawhide' || matrix.continue-on-error == true }}
```

Comment:
```suggestion
# FIXME
    continue-on-error: ${{ matrix.distro == 'opensuse:latest' }}
```
All other distros work, don't they?

Reply:
`fedora:rawhide` and `ubuntu:devel` are a bit flacky at times hence I added them here.